hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/rollout/RolloutListGrid.java

```diff
@@ -132,6 +132,8 @@
                 uiNotification, uiProperties, entityFactory, i18n, eventBus, targetFilterQueryManagement);
         this.uiNotification = uiNotification;
         this.rolloutUIState = rolloutUIState;
+
+        init();
```

Comment:
We do this practically in every UI class. But you are right. This is dangerous and should be refactored.